bibtex/working-group.bib

```diff
@@ -0,0 +1,595 @@
+	month = {July},
+	year = {2017}
+}
+
+@unpublished{steeves_organizational_2015,
+	address = {Washington, D.C},
+	title = {Organizational {Implications} of {Data} {Science} {Environments} in {Education}, {Research}, and {Research} {Management} in {Libraries}},
+	url = {https://osf.io/rhgq3/},
+	abstract = {A presentation by Vicky Steeves, Erik Mitchell, and Jennifer Muilenburg on the implications of data science in a library environment.},
+	language = {en},
+	note = {CNI Fall Meeting},
+	author = {Steeves, Vicky and Mitchell, Erik and Muilenburg, Jennifer},
+	month = {December},
+	year = {2015}
+}
+
+@misc{group_reproducible_2017,
+	title = {Reproducible {Science} @ {NYU}},
+	url = {https://nyu.reproduciblescience.org},
+	abstract = {A website that gives an overview to the work of the NYU Open Science and Reproducibility Working Group.},
+	language = {en},
+	journal = {Reproducible Science @ NYU},
+	author = {Group, NYU Reproducibility Working},
+	month = {January},
+	year = {2017}
+}
+
+@misc{group_reproducible_2018,
+	title = {Reproducible {Science}},
+	url = {https://reproduciblescience.org/},
+	abstract = {On this site, the moderators have curated sources of various types discussing reproducibility in the directory. You can find academic papers, blog posts, popular media articles, talks, tools, and more.},
+	language = {en},
+	journal = {Reproducible Science},
+	author = {Group, NYU Reproducibility Working},
+	month = {June},
+	year = {2018}
+}
+
+@misc{steeves_how_2017,
+	title = {How {GitLab} can help in research reproducibility},
+	url = {https://about.gitlab.com/2017/08/25/gitlab-and-reproducibility/},
+	abstract = {NYU reproducibility librarian Vicky Steeves shares why GitLab is her choice for ongoing collaborative research, and how it can help overcome challenges with sharing code in academia.},
+	language = {en},
+	type = {Blog},
+	journal = {GitLab},
+	author = {Steeves, Vicky},
+	month = {August},
+	year = {2017}
+}
+
+@misc{steeves_reproducibility-news_2018,
+	title = {reproducibility-news},
+	shorttitle = {reproducibility-news},
+	url = {https://github.com/ViDA-NYU/reproducibility-news},
+	abstract = {An RSS feed of curated items relating to reproducibility.},
+	year={2018},
+	publisher = {ViDA-NYU},
+	author = {Steeves, Vicky and Rampin, Rémi},
+	month = {June},
+	year = {2018}
+}
+
+@misc{rampin_repromatch_nodate,
+	title = {{ReproMatch}},
+	url = {http://repromatch.poly.edu/tools/search/},
+	abstract = {Many tools have been developed to support researchers in the practice of reproducible research. However, given the many requirements across disciplines, identifying the right tool or a set of tools can be a daunting task. With the help of developers of reproducibility tools, we will create a comprehensive catalog of tools. Using ReproMatch, users will be able to search for tools that match their reproducibility needs.},
+	year={2018},
+	journal = {ReproMatch},
+	author = {Rampin, Rémi and Ellqvist, Tommy}
```

Comment:
I didn't work on ReproMatch, you can see the team on http://repromatch.poly.edu/about/

It's at least 3 years old, @fchirigati?

Reply:
Ah, sorry, I went on the GitHub contributors (it's just you and Tommy, @remram44).

Reply:
Yep, we released the website on March 2013.

Reply:
Ok, will fix!